tests/committable.py

```diff
@@ -50,33 +50,72 @@ def run(args):
         backups[2].suspend()
         backups[3].stop()
 
-        txs = []
-        # Run some transactions that can't be committed
+        committable_txs = []
+        # Run some transactions that can't be committed now
         with primary.client("user0") as uc:
             for i in range(3):
-                txs.append(
+                committable_txs.append(
                     uc.post("/app/log/private", {"id": 100 + i, "msg": "Hello world"})
                 )
 
-        sig_view, sig_seqno = txs[-1].view, txs[-1].seqno + 1
+        last_tx = committable_txs[-1]
+        sig_view, sig_seqno = last_tx.view, last_tx.seqno + 1
         with backups[0].client() as bc:
             wait_for_pending(bc, sig_view, sig_seqno)
 
-        # Kill the primary, restore other backups
-        primary.stop()
+        # Suspend the final backup and run some transactions which only the primary hears, which should be discarded by the new primary
+        backups[0].suspend()
+        uncommittable_txs = []
```

Comment:
I suggest a different name, because we call a committable transaction anything that has a valid signature after it. Maybe post_partition_txs or something like that.